examples/tokenvm/actions/common.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package actions
+
+const (
+	burnAssetID uint8 = iota
```

Comment:
Please list these out explicitly. `iota` will do the job but find it is much more delicate.

Reply:
If there is duplicated id created manually, the registry will return the error 👍

Reply:
May be worth adding that as a comment just for other contributors to keep in mind.